test/e2e/specs/onboarding/ftme__site-assembler.ts

```diff
@@ -118,14 +118,18 @@ describe( 'Site Assembler', () => {
 			await siteAssemblerFlow.clickButton( 'Save and continue' );
 		} );
 
-		it( 'Click "Save and continue" and land on the Site Editor', async function () {
-			const urlPromise = page.waitForURL( /processing/ );
+		it( 'Click "Start adding content" to the next step', async function () {
 			await siteAssemblerFlow.clickButton( 'Start adding content' );
-			await urlPromise;
+		} );
+
+		it( 'Land on the Celebration step', async function () {
+			await page.waitForURL( /processing/ );
+			await page.waitForURL( /celebration-step/, { timeout: 30 * 1000 } );
```

Comment:
**suggestion:non-blocking** - I ran the spec locally where the redirection to `/celebration-step/` took more than 30s. 

I think it might be useful to have this timeout be set to a longer value.

Reply:
Two runs both failed:

```
  ● Site Assembler › Site Assembler › Land on the Celebration step

    page.waitForURL: Timeout 30000ms exceeded.
    =========================== logs ===========================
    waiting for navigation until "load"
    ============================================================

      125 | 		it( 'Land on the Celebration step', async function () {
      126 | 			await page.waitForURL( /processing/ );
    > 127 | 			await page.waitForURL( /celebration-step/, { timeout: 30 * 1000 } );
          | 			           ^
      128 | 		} );
      129 |
      130 | 		it( 'Click "Edit your content" and land on the Site Editor', async function () {

      at Object.waitForURL (specs/onboarding/ftme__site-assembler.ts:127:15)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 skipped, 11 passed, 13 total
Snapshots:   0 total
Time:        59.184 s, estimated 120 s
```